test/e2e/suite/cases/pkcs12.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cases
+
+import (
+	"context"
+	"time"
+
+	"github.com/cert-manager/cert-manager/pkg/util/pki"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"software.sslmate.com/src/go-pkcs12"
+
+	"github.com/cert-manager/csi-driver/test/e2e/framework"
+)
+
+var _ = framework.CasesDescribe("Should write keystore pkcs12 file correctly", func() {
+	f := framework.NewDefaultFramework("pkcs12")
+
+	It("should create a pod with the pkcs12 file written", func() {
+		testVolume, testPod := basePod(f, map[string]string{
+			"csi.cert-manager.io/issuer-name":              f.Issuer.Name,
```

Comment:
**issue (formatting, blocking):** This file needs `go fmt`, as it improves readability for these map literals